osu.Game.Rulesets.Mania/Edit/ManiaBeatSnapGrid.cs

```diff
@@ -107,21 +107,21 @@ private void createLines()
             if (selectionTimeRange == null)
                 return;
 
-            var range = selectionTimeRange.Value;
+            var (start, end) = selectionTimeRange.Value;
```

Comment:
This is also arguably worse what with the `start - time` expressions below. Should be `(rangeStart, rangeEnd)` if we *really* want the deconstruction imo. Which I'm not super sold on in the first place.